pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "esque"
-version = "0.2.2"
+version = "0.2.3a1"
```

Comment:
I think we don't need an alpha version for this. Since we're still on major version zero, just a patch bump is fine.